src/python/CRABInterface/RESTTask.py

```diff
@@ -115,6 +115,16 @@ def webdir(self, **kwargs):
         yield row.user_webdir
 
 
+    def getpublishurl(self, **kwargs):
+        if 'workflow' not in kwargs or not kwargs['workflow']:
+            raise InvalidParameter("Task name not found in the input parameters")
+        workflow = kwargs['workflow']
+        try:
+            row = next(self.api.query(None, None, self.Task.GetPublishUrl_sql, taskname=workflow))
+        except StopIteration:
+            raise ExecutionError("Impossible to find task %s in the database." % kwargs["workflow"])
+        return row
```

Comment:
Single or double quotes make no difference. You're right that it should be consistent, though it's not something that has a general convention and in our code both styles are used quite randomly. Up to @dciangot